src/modules/account/submodule/multisig/views/ViewAccountMultisigEditAccount.vue

```diff
@@ -272,8 +272,12 @@ export default {
       }
       let name = []
       cosignaturies.value.forEach(publicKey=>{
+        let address = Address.createFromPublicKey(publicKey,AppState.networkType).plain()
+        let contact = walletState.currentLoggedInWallet.contacts.find((contact) => contact.address==address);
         if(wallet.accounts.find(acc=>acc.publicKey ===publicKey)){
           name.push(wallet.accounts.find(acc=>acc.publicKey ==publicKey).name)
+        }else if(contact){
+          name.push(contact.name)
```

Comment:
code is correct, but sequence wrong, displaying contact name has a higher priority. So, if contact,..., else if account.find,...,else...

Reply:
noted with thanks, will make changes based on the comment

Reply:
done revising the sequence